test/test_framework_test.go

```diff
@@ -2757,6 +2757,159 @@ func TestReplaceImports(t *testing.T) {
 	assert.Equal(t, expected, replacedCode)
 }
 
+func TestServiceAccount(t *testing.T) {
+	t.Parallel()
+
+	t.Run("retrieve from EmulatorBackend", func(t *testing.T) {
+		t.Parallel()
+
+		emulatorBackend := NewEmulatorBackend(nil, nil, nil)
+
+		serviceAccount, err := emulatorBackend.ServiceAccount()
+
+		require.NoError(t, err)
+		assert.Equal(
+			t,
+			"0xf8d6e0586b0a20c7",
+			serviceAccount.Address.HexWithPrefix(),
+		)
+	})
+
+	t.Run("retrieve from Test framework's blockchain", func(t *testing.T) {
+		t.Parallel()
+
+		const testCode = `
+		    import Test
+
+		    pub let blockchain = Test.newEmulatorBlockchain()
+
+		    pub fun testGetServiceAccount() {
+		        // Act
+		        let account = blockchain.serviceAccount()
+
+		        // Assert
+		        Test.assert(account.address.getType() == Type<Address>())
+		        Test.assert(account.publicKey.getType() == Type<PublicKey>())
+		        Test.assert(account.address == 0xf8d6e0586b0a20c7)
+		    }
+		`
+
+		runner := NewTestRunner()
+
+		result, err := runner.RunTest(testCode, "testGetServiceAccount")
+		require.NoError(t, err)
+		require.NoError(t, result.Error)
+	})
+
+	t.Run("run scripts and transactions with service account", func(t *testing.T) {
+		t.Parallel()
+
+		const testCode = `
+		    import Test
+
+		    pub let blockchain = Test.newEmulatorBlockchain()
+
+		    pub fun testGetServiceAccountBalance() {
+		        // Arrage
```

Comment:
```suggestion
		        // Arrange
```

Reply:
Fixed in https://github.com/onflow/cadence-tools/pull/122/commits/4144d8e86d616b66d4277461f5484b8c81333610